extensions/amp-experiment/1.0/test/test-attribute-mutation.js

```diff
@@ -75,71 +76,105 @@ describes.realWin(
+        it('forbid keywords', () => {
+          // !important
+          let attributeMutation = getAttributeMutationDefaultStyle(
+            'color: #000000 !important;'
           );
-          expect(attributeMutation.validate()).to.be.equal(true);
-        });
+          expect(attributeMutation.parseAndValidate()).to.be.equal(false);
 
-        it('should allow background-color mutations', () => {
-          const attributeMutation = getAttributeMutationDefaultStyle(
-            'background-color: #000000'
+          // <
+          attributeMutation = getAttributeMutationDefaultStyle(
+            'color: <#000000;'
           );
-          expect(attributeMutation.validate()).to.be.equal(true);
+          expect(attributeMutation.parseAndValidate()).to.be.equal(false);
         });
 
-        it('should allow color mutations', () => {
+        it('NOT allowed style mutations', () => {
           const attributeMutation = getAttributeMutationDefaultStyle(
-            'color: #000000'
+            'random: abc'
           );
-          expect(attributeMutation.validate()).to.be.equal(true);
+          expect(attributeMutation.parseAndValidate()).to.be.equal(false);
         });
 
-        it('should not allow !important mutations', () => {
-          const attributeMutation = getAttributeMutationDefaultStyle(
-            'color: #000000 !important;'
+        it('style mutations format', () => {
+          let attributeMutation = getAttributeMutationDefaultStyle('');
+          expect(attributeMutation.parseAndValidate()).to.be.equal(true);
+
+          attributeMutation = getAttributeMutationDefaultStyle(
+            'background-color red'
+          );
+          expect(attributeMutation.parseAndValidate()).to.be.equal(false);
+
+          attributeMutation = getAttributeMutationDefaultStyle(
+            'background-color ::red'
+          );
+          expect(attributeMutation.parseAndValidate()).to.be.equal(false);
+
+          attributeMutation = getAttributeMutationDefaultStyle(
+            'background-color: '
           );
-          expect(attributeMutation.validate()).to.be.equal(false);
+          expect(attributeMutation.parseAndValidate()).to.be.equal(true);
+
+          attributeMutation = getAttributeMutationDefaultStyle(
+            'background-color: red; ;'
+          );
+          expect(attributeMutation.parseAndValidate()).to.be.equal(true);
+
+          attributeMutation = getAttributeMutationDefaultStyle(
+            '  background-color: red      ;background-color:green;   '
+          );
+          expect(attributeMutation.parseAndValidate()).to.be.equal(true);
         });
 
-        it('should not allow HTML Comment mutations', () => {
-          const attributeMutation = getAttributeMutationDefaultStyle(
-            '<!-- color: #000000;'
+        it('background-color mutation', () => {
+          let attributeMutation = getAttributeMutationDefaultStyle(
+            'background-color: #000000'
```

Comment:
use a color with letter number combination for better test coverage

Reply:
SG. However I also don't think `#000000` has bad test coverage. If you look at the code, we allow anything for `background-color`. To me, `#000000` and `#0123df` makes no difference here.

Reply:
done